agreement_sale/models/sale.py

```diff
@@ -0,0 +1,14 @@
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    agreement_id = fields.Many2one(
```

Comment:
@bealdav  Can I remove [agreement_id](https://github.com/OCA/contract/pull/294/files#diff-c4b698254a164c50c4c83c3cb1965539R14) field from my module? (agreement_legal_sale)
If yes then can you please pass copy=False attribute on this field.
Also, This PR is ready for review or in WIP?

@max3903

Reply:
Done

Reply:
@murtuzasaleh Yes you can.